volto/src/helpers/index.js

```diff
@@ -18,9 +18,9 @@ const replaceUrl = (originalPath, parentPath) => {
   if (!originalPath) {
     return originalPath;
   }
-  const dashboard_name = parentPath.match(/(?<=\/dashboards\/).*./);
+  const dashboard_name = parentPath.match(/(?:\/dashboards\/).*./);
   const possible_url = originalPath.replace(
-    /(?<=\/dashboards\/).+?(?=\/)/,
```

Comment:
You're right, the lookbehind doesn't capture `/dashboards/`, but I was trying to avoid the lookbehind function completely.

In context it shouldn't matter if we are also replacing dashboard as well as it should always be the same part getting replaced.